### PR TITLE
[MNG-7079] Upgrade to jansi 2.2.0

### PR DESCRIPTION
**Pull request by gnodet**
This PR depends on https://github.com/apache/maven-shared-utils/pull/69 and the release of `maven-shared-utils`.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -319,12 +319,12 @@ under the License.
       <dependency>
         <groupId>org.apache.maven.shared</groupId>
         <artifactId>maven-shared-utils</artifactId>
-        <version>3.2.1</version>
+        <version>3.3.4-SNAPSHOT</version>
```
Comment (elharo) [inline pom.xml:322]: next release of maven-shared-utils should probably be 3.4.0 since you've added new public API and features.

Comment (gnodet) [reply pom.xml:322]: I'll upgrade the PR with the released version once https://github.com/apache/maven-shared-utils/pull/69 is merged and releases, whatever the exact version is.

Comment (MartinKanters) [reply pom.xml:322]: @gnodet Took some time, but we managed to release maven-shared-utils 3.3.4 yesterday. I believe [your PRs](https://github.com/apache/maven/pulls/gnodet) are now unblocked.

**Review by olamy — APPROVED** [review]

**Comment by MartinKanters** [comment]
Testing on Jenkins: https://ci-builds.apache.org/job/Maven/job/maven-box/job/maven/job/MNG-7079/1/

**Comment by MartinKanters** [comment]
Thanks for the contribution @gnodet! I can probably take at your other PRs this evening.

**Comment by jira-importer** [comment]
Resolve #8378

**Pull request merged**